code/modules/reagents/newchem/other.dm

```diff
@@ -117,6 +117,12 @@ datum/reagent/colorful_reagent
 	result_amount = 6
 	mix_message = "The substance flashes multiple colors and emits the smell of a pocket protector."
 
+datum/reagent/colorful_reagent/reaction_mob(var/mob/living/simple_animal/M, var/method=TOUCH, var/volume)
+    if(M)
```

Comment:
I know, but it won't detect the corgi file for some reason. Unknown type,
it says.